openstack/migrate-helpers/security_group_exporter.rb

```diff
@@ -0,0 +1,56 @@
+#
+# security_group_exporter exports all of the security groups and creates a script to re-import them (most likely into a new openstack deployment).
+#
+# Note: As of now, the script just ignores the `default` security group.
+#
+# Original Author: AJ Bahnken / ajvb
+#
+
+require 'json'
+
+# Create output folder
+todays_date = Time.now.strftime "%Y-%m-%d"
+system 'mkdir', "secgroup_exporter_#{todays_date}"
+
+secgroup_importer_script = File.new("secgroup_exporter_#{todays_date}/secgroup_importer.sh", "w")
```

Comment:
Non-Blocking: If we wanted this to be a little prettier, you could use the File.open block syntax:
```
File.open("new_file", "w") do |script|
  script.puts "My stuff"
end
```

Which will also close the file at the end of the block.